api/proxy/store/builder/storage_manager_builder.go

```diff
@@ -580,6 +593,10 @@ func buildPayloadDisperser(
 		return nil, fmt.Errorf("new disperser client: %w", err)
 	}
 
+	if metrics != nil {
+		disperserClient.SetClientMetrics(metrics)
+	}
```

Comment:
In order to get the probe metrics to show up in proxy, you will need to pass in a registry to the NewPayloadDisperser function below, instead of nil